KBaseSearchEngine.spec

```diff
@@ -174,6 +174,16 @@ module KBaseSearchEngine {
         mapping<string, string> object_props;
     } ObjectData;
 
+    typedef int accessGroupId;
+    typedef string timestamp;
+
+    typedef tuple<string narrative_name,
+                  int narrative_id,
```

Comment:
to be more consistent with their actual origin, perhaps
narrative_id -> workspace_id: since it is really the workspace id; new readers might think there actually is a narrative id.

for completeness wouldn't you also want the object_id? 

that would allow us to create narrative links from NarrativeInfo.

Reply:
The narrative_id is not the workspace_id. The narrative_id is the object_id of the narrative.